Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -56,7 +56,7 @@
         "EditableOptions": "True",
         "DisableManageLink": "True"
       },
-      "helpMarkDown": "Select the build definition name"
+      "helpMarkDown": "Select the build definition ID"
```

Comment:
Task takes name as input but uses ID internally. So the current error message is good.